x-pack/plugins/cases/server/connectors/case/schema.ts

```diff
@@ -56,19 +56,30 @@ const ResilientFieldsSchema = schema.object({
   severityCode: schema.nullable(schema.string()),
 });
 
-const ServiceNowFieldsSchema = schema.object({
+const ServiceNowITSMFieldsSchema = schema.object({
   impact: schema.nullable(schema.string()),
   severity: schema.nullable(schema.string()),
   urgency: schema.nullable(schema.string()),
   category: schema.nullable(schema.string()),
   subcategory: schema.nullable(schema.string()),
 });
 
+const ServiceNowSIRFieldsSchema = schema.object({
```

Comment:
Just curious but in the actions plugin the schema looks like this:

```
// Schema for ServiceNow Security Incident Response (SIR)
export const ExecutorSubActionPushParamsSchemaSIR = schema.object({
  incident: schema.object({
    ...CommonAttributes,
    dest_ip: schema.nullable(schema.string()),
    malware_hash: schema.nullable(schema.string()),
    malware_url: schema.nullable(schema.string()),
    source_ip: schema.nullable(schema.string()),
    priority: schema.nullable(schema.string()),
  }),
  comments: CommentsSchema,
});
```

Why is the schema there all strings? Is it because the values are being converted to strings when they're placed in the saved object?

Reply:
@jonathan-buttner Hey! Good question. At the moment the SIR connector is being used only by Cases which has its own use case. For Alerts & Actions it is not known yet how it will be used so for the moment it follows ServiceNow SIR schema.